src/policy/copyspace.rs

```diff
@@ -188,6 +188,11 @@ impl<VM: VMBinding> CopySpace<VM> {
     }
 
     pub fn release(&self) {
+        #[cfg(feature = "poison_on_release")]
+        for (start, size) in self.pr.iterate_allocated_regions() {
+            crate::util::memory::set(start, 0xbc, size);
```

Comment:
Instead of using an arbitrary value, you can define a mapping between space and its poison value, or use something from the space (such as the SpaceDescriptor index, or the space name).

I would suggest creating a module `crate::util::poison_on_release`. Define the poison values there, and define functions to poison a memory region.

Reply:
I am not sure if we need the different values to tell which space it is, especially after Kunshan's change in https://github.com/mmtk/mmtk-core/pull/1236. But if we do want to use different values, there needs to be a place where we can look at and know what the value means.